internal/sidebar/direct/channel.go

```diff
@@ -28,6 +28,7 @@ type Channel struct {
 
 var channelCSS = cssutil.Applier("direct-channel", `
 	.direct-channel {
+		margin-left: 6px;
```

Comment:
You might not want to use a `margin-left` here. Consider adding into `padding` instead: `padding: 4px 12px;`.

Reply:
Done